File: src/main/java/ar/edu/um/tesoreria/rest/model/TipoPago.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.model;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table(name = "tipopago")
@EqualsAndHashCode(callSuper=false)
@NoArgsConstructor
@AllArgsConstructor
public class TipoPago extends Auditable implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = -4739977530340335996L;

	@Id
	@Column(name = "tpa_id")
	private Integer tipoPagoId;
	
	@Column(name = "tpa_nombre")
	private String nombre = "";

}

File: src/main/java/um/tesoreria/rest/kotlin/model/TipoPago.kt
package um.tesoreria.rest.kotlin.model

import um.tesoreria.rest.model.Auditable
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "tipopago")
data class TipoPago(

    @Id
    @Column(name = "tpa_id")
    var tipoPagoId: Int? = null,

    @Column(name = "tpa_nombre")
    var nombre: String = "",

    ) : Auditable()
